File: cs/Iris/ParserUtils/LexParseException/UnfinishedCommentary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    class UnfinishedCommentary : BaseException
    {
        public UnfinishedCommentary(int lineNum, int colNum)
            : base(lineNum, colNum)
        {

        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append("unfinished commentary: started at ");
            str.Append(base.stringTrace());

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/UnfinishedCommentary.java
package com.rokuan.iris.exception;

public class UnfinishedCommentary extends BaseException
{
	public UnfinishedCommentary(int lineNum, int colNum)
	{
		super(lineNum, colNum);
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append("unfinished commentary: started at ");
		str.append(super.stringTrace());

		return str.toString();
	}
}
